src/Components/OATPropertyEditor/PropertyList.tsx

```diff
@@ -67,8 +81,15 @@ export const PropertyList = ({
         propertySelectorTriggerElementsBoundingBox,
         setPropertySelectorTriggerElementsBoundingBox
     ] = useState(null);
+    const [propertyListItems, setPropertyListItems] = useState<
+        ICardboardListItem<IOATProperty>[]
+    >([]);
     const { model, templates } = state;
 
+    useEffect(() => {
+        getListItems(listItemOnClick);
+    }, [propertyList]);
```

Comment:
Could we create propertiesListItems as a `useMemo` instead?

It looks like its value is determined by changes to `propertyList` only?